docs/api/en/objects/InstancedMesh.html

```diff
@@ -53,6 +53,23 @@ <h3>[property:BufferAttribute instanceMatrix]</h3>
 		<h2>Methods</h2>
 		<p>See the base [page:Mesh] class for common methods.</p>
 
+		<h3>[method:null getMatrixAt]( [param:Integer index], [param:Matrix4 matrix] )</h3>
+		<p>
+			[page:Integer index]: The index of an instance. Values have to be in the range [0, count].
+		</p>
+		<p>
+			[page:Matrix4 matrix]: This 4x4 matrix will be set to the local transformation matrix of the defined instance.
+		</p>
+		<p>
+			Get the local transformation matrix of the defined instance.
+		</p>
+
+		<h3>[method:null raycast]( [param:Raycaster raycaster], [param:Array intersects] )</h3>
```

Comment:
This block can be deleted since `raycast()` is already documented in upper classes.

Reply:
OK, I'll change it:smile: